tensorboard/webapp/runs/types.ts

```diff
@@ -61,16 +61,10 @@ export enum GroupByKey {
   REGEX,
 }
 
-export interface BaseGroupBy {
-  key: GroupByKey.EXPERIMENT | GroupByKey.RUN;
-}
-
-export interface RegexGroupBy {
-  key: GroupByKey.REGEX;
-  regexString: string;
-}
-
-export type GroupBy = BaseGroupBy | RegexGroupBy;
+export type GroupBy = {
+  key: GroupByKey | null;
```

Comment:
`key` should not be nullable. Please consider reverting back this change. It was strongly typed as is.